ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,405 @@
+                sizeCategoryChanged = true
+                setupLayoutConstraints()
+            }
+            if #available(iOS 13, *) {
+                if previousTraitCollection.hasDifferentColorAppearance(comparedTo: traitCollection) {
+                    // Update border color
+                    switch colorTheme {
+                    case .appColor:
+                        if let window = window {
+                            layer.borderColor = UIColor(light: Colors.primaryTint30(for: window), dark: .clear).cgColor
+                        }
+                    case .neutral:
+                        layer.borderColor = Constants.defaultBorderColor.cgColor
+                    case .custom:
+                        layer.borderColor = customBorderColor.cgColor
+                    }
+                }
+            }
+        }
+    }
+
+    /// Set up the background color of the card and update the icon and text color if necessary
+    private func setupColors() {
+        switch colorTheme {
+        case .appColor:
+            primaryLabel.textColor = UIColor(light: .black, dark: Colors.gray100)
+            secondaryLabel.textColor = UIColor(light: Colors.gray600, dark: Colors.gray400)
+            iconView.tintColor = UIColor(light: Colors.gray600, dark: Colors.gray500)
+            if let window = window {
+                backgroundColor = UIColor(light: Colors.primaryTint40(for: window), dark: Colors.primaryTint30(for: window))
+                layer.borderColor = UIColor(light: Colors.primaryTint30(for: window), dark: .clear).cgColor
+            }
+        case .neutral:
+            backgroundColor = Constants.defaultBackgroundColor
+            primaryLabel.textColor = Constants.defaultTitleColor
+            secondaryLabel.textColor = Constants.defaultSubtitleColor
+            iconView.tintColor = Constants.defaultIconTintColor
+            layer.borderColor = Constants.defaultBorderColor.cgColor
+        case .custom:
+            backgroundColor = customBackgroundColor
+            primaryLabel.textColor = customTitleColor
+            secondaryLabel.textColor = customSubtitleColor
+            iconView.tintColor = customIconTintColor
+            layer.borderColor = customBorderColor.cgColor
+        }
+    }
+
+    private struct Constants {
+        static let defaultBackgroundColor = UIColor(light: .white, dark: Colors.gray900)
+        static let defaultBorderColor = UIColor(light: Colors.gray100, dark: .clear)
+        static let defaultTitleColor = UIColor(light: Colors.gray900, dark: Colors.gray100)
+        static let defaultSubtitleColor = UIColor(light: Colors.gray500, dark: Colors.gray400)
+        static let defaultIconTintColor = UIColor(light: Colors.gray400, dark: Colors.gray500)
+        static let iconSize = CGSize(width: 24, height: 24)
+        static let borderWidth: CGFloat = UIScreen.main.devicePixel
```

Comment:
This should be lazy, not static because you don't want UIScreen.main.devicePixel to get called when this component is loaded into the program. I would recommend either getting rid of this constant and just calling UIScreen.main.devicePixel or using a lazy var.